setup.py

```diff
@@ -690,7 +690,7 @@ def _read_requirements(filename: str) -> list[str]:
         "tensorizer": ["tensorizer>=2.9.0"],
         "runai": ["runai-model-streamer", "runai-model-streamer-s3", "boto3"],
         "audio": ["librosa", "soundfile"],  # Required for audio processing
-        "video": ["decord"]  # Required for video processing
+        "video": []  # decord is deprecated
```

Comment:
With `decord` gone, this comment might be confusing for future devs who didn't know we ever used it. How about this:

```suggestion
        "video": []  # Kept for backwards compatability
```